.github/workflows/workflow.yml

```diff
@@ -3,6 +3,7 @@ name: Checks and Tests
 on:
   push:
     branches:
+      - develop
```

Comment:
should you not remove master from the line below?

Reply:
I've re-enabled that for master. The reason: we run this set of checks for each PR and push to develop/master. For release PR I've decided not to repeat this checks but still run it and plus additional pre-release checks (for now integration tests on macOS).